extensions/youtube/src/lib/youtubeapi.ts

```diff
@@ -127,6 +127,11 @@ export interface Channel {
   relatedPlaylists?: ChannelRelatedPlaylists;
 }
 
+export interface ChannelActionProps {
```

Comment:
Since the props interfaces are not part of the business logic of the youtube API please move it to it's own file

Reply:
Moved to `actions.tsx`